Description: Disable relocation-model=static test for armhf, failing on armhf
Author: Ximin Luo <infinity0@debian.org>
Bug: https://github.com/rust-lang/rust/issues/33809
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/test/run-make/relocation-model/Makefile
+++ b/src/test/run-make/relocation-model/Makefile
@@ -12,6 +12,9 @@
 ifdef IS_MSVC
 # FIXME(#28026)
 others:
+else ifeq ($(filter arm,$(LLVM_COMPONENTS)),arm)
+# FIXME(#33809)
+others:
 else
 others:
 	$(RUSTC) -C relocation-model=static foo.rs
